File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int n=1,i=0,j=i+1;
        while(j!=nums.size()){
            if(nums[j]==nums[i]) j++;
            else{
                i++;
                swap(nums[i],nums[j]);
                j++;
                n++;
            }
        }
        return n;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: list[int]) -> int:
        i=0
        j=1
        n=1
        if(len(nums)==1): return n
        while(j<len(nums)):
            if(nums[i]==nums[j]):
                j+=1
            else:
                i+=1
                nums[i],nums[j]=nums[j],nums[i]
                j+=1
                n+=1
        return n
